lua/persisted/init.lua

```diff
@@ -12,23 +12,18 @@ function M.fire(event)
   vim.api.nvim_exec_autocmds("User", { pattern = "Persisted" .. event })
 end
 
----Get the current working directory
----@return string
-function M.cwd()
-  return utils.sanitize_dir(vim.fn.getcwd())
-end
-
 ---Get the current session for the current working directory and git branch
 ---@param opts? {branch?: boolean}
 ---@return string
 function M.current(opts)
   opts = opts or {}
-  local name = M.cwd()
+  local name = utils.sanitize_dir(vim.fn.getcwd())
 
   if config.use_git_branch and opts.branch ~= false then
     local branch = M.branch()
     if branch then
-      name = name .. "@@" .. branch:gsub("[\\/:]+", "%%")
+      branch = utils.sanitize_dir(branch)
```

Comment:
Shall we rename it from `sanitize_dir` as we're using it on a branch?

Reply:
I was thinking about that, I can rename. What do you think is a good name? The sanitizing is for being path-safe, some ideas I have are
- `sanitize_pathname`
- `make_fs_safe`

Reply:
`make_fs_safe` gets be excited!

Reply:
Awesome, done!